### DISCUSSION TITLE
dunglas/symfony-docker : Internal call to same project

### DISCUSSION THREAD

**Discussion by fxlacroix** [dunglas/symfony-docker#214] [Q&A]
I've just tried https://github.com/dunglas/symfony-docker/ 
It seems very cool and work on the road !

But i can't ping my server sf from sf... i'm trying to call my own api in the same project but the ping localhost from sf does not seem to work.

I've put a call.php and a receipt.php in public/, but i don't know what to do so that call.php ping receipt.php I can access to https://localhost/call.php but call.php can't access https://localhost/receipt.php from my php script. i can access https://localhost/receipt.php from my navigator.

if somebody have a clue ? ty.

**Comment by freiondrej-lmc** [comment]
Hello @fxlacroix , when I tried creating a call.php file in public/ and navigating to https://localhost/call.php, I got to the symfony app (public/index.php) with a 404 saying there is no route for "call.php". The server is probably configured so that all requests are going to index.php and Symfony then handles the routing internally. Are you able to reach call.php or receipt.php contents? I suppose those should rather be Symfony controllers with respective routes in them.
